hi-ml-cpath/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -809,7 +811,7 @@ def test_encoder_checkpointning(
     # 1. Compare the loss and gradients of the encoder with and without checkpointing
     validate_loss_with_activations_checkpointing(train_dataloader, model_ckpt_enc, model_no_ckpt_enc, encoder_type)
 
-    if encoder_type != SwinTransformer_NoPreproc.__name__:  # SwinT requires images of 224 input size, mock tiles are 28
+    if "Resnet" in encoder_type:  # SwinT and DenseNet requires images of 224 input size, mock tiles are 28
```

Comment:
Ah I see, I guess it is the same reason as the above check (line 768). define once above

Reply:
yes exactly